src/adapter/src/coord.rs

```diff
@@ -788,10 +783,39 @@ impl<S: Append + 'static> Coordinator<S> {
         }
     }
 
-    #[allow(dead_code)]
-    async fn create_storage_export(&mut self, id: GlobalId, sink: &Sink) {
+    async fn create_storage_export(
```

Comment:
This method probably belongs in `ddl.rs`. I think I just missed it in the refactor because of the `dead_code`.